protobuf/as/integration/integration.proto

```diff
@@ -169,6 +169,15 @@ message TxAckEvent {
 
     // User-defined device tags.
     map<string, string> tags = 6;
+
+    // Gateway ID.
+    bytes gateway_id = 7 [json_name = "gatewayID"];
+	
+	// Gateway name.
```

Comment:
Would you mind removing `gateway_name`? None of the Protobuf messages contain the `gateway_name`, it is better to keep this consistent and it removes the requirement to do an other lookup (gateway id to gateway name).

Reply:
I have resolved your review comemnt